src/outputs/output_utils.cpp

```diff
@@ -125,10 +125,11 @@ int VarInfo::GetDim(int i) const {
 }
 
 std::vector<VarInfo> VarInfo::GetAll(const VariableVector<Real> &vars,
-                                     const IndexShape &cellbounds) {
+                                     const IndexShape &cellbounds,
+                                     const IndexShape &f_cellbounds) {
   std::vector<VarInfo> out;
   for (const auto &v : vars) {
-    out.emplace_back(v, cellbounds);
+    out.emplace_back(v, v->IsSet(Metadata::Fine) ? f_cellbounds : cellbounds);
```

Comment:
here and below: Where is the info on `f_cellbounds` in the context of `VarInfo` actually being used, i.e., why is this change necessary?

Reply:
I think this eventually gets used to set the size of the datasets that are output to the hdf5 file, so if I try to output a `Fine` field without this change it would only write $N^3$ points instead of $(2N)^3$ points.